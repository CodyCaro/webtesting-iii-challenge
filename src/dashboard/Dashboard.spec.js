// Test away!
import React from "react";
import { render, shallow } from "react-testing-library";
import Dashboard from "./Dashboard";

test("Gate defaults to Unlocked and Open", () => {
  const wrapper = render(<Dashboard />);
  expect(wrapper.baseElement).toMatchSnapshot();
});
