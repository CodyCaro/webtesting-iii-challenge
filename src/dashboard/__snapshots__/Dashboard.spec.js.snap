// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gate defaults to Unlocked and Open 1`] = `
<body>
  <div>
    <div
      class="display panel"
    >
      <div
        class="led green-led"
        data-testid="locked-unlocked-display"
      >
        Unlocked
      </div>
      <div
        class="led green-led"
        data-testid="open-close-display"
      >
        Open
      </div>
    </div>
    <div
      class="controls panel"
    >
      <button
        class="toggle-btn"
        data-testid="locked-unlocked"
        disabled=""
      >
        Lock Gate
      </button>
      <button
        class="toggle-btn"
        data-testid="open-close"
      >
        Close Gate
      </button>
    </div>
  </div>
</body>
`;
